test/unit/serverConfigParams.test.ts

```diff
@@ -0,0 +1,215 @@
+        config.util.extendDeep(config, {
+          app: {
+            fixedUsdFee: 0,
+            gasFeePercentage: 0,
+            transferFeePercentage: -1,
+          },
+        });
+
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should pass if only gasFeePercentage is properly configured', function () {
+        config.util.extendDeep(config, { app: { gasFeePercentage: 0.1 } });
+
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should pass if only transferFeePercentage is properly configured', function () {
+        config.util.extendDeep(config, {
+          app: { transferFeePercentage: 0.01 },
+        });
+
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should pass if only fixedUsdFee is properly configured', function () {
+        config.util.extendDeep(config, { app: { fixedUsdFee: 1 } });
+
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should pass if more than one fee is properly configured', function () {
+        config.util.extendDeep(config, {
+          app: { fixedUsdFee: 1, gasFeePercentage: 0.1 },
+        });
+
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+    });
+
+    describe('Custom functions on /serverConfigParamsUtils.ts', function () {
+      it('Should fail if the string representation of a BigNumber is incorrect', function () {
+        config.util.extendDeep(config, {
+          blockchain: { workerMinBalance: '1e10' },
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          '"blockchain.workerMinBalance" failed custom validation because Invalid BigNumber string representation: 1e10'
+        );
+      });
+
+      it('Should pass if the string representation of a BigNumber is correct', function () {
+        config.util.extendDeep(config, {
+          blockchain: {
+            workerMinBalance: '1035386848354684654698546564654654',
+          },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
+      });
+
+      it('Should fail if an address is incorrect', function () {
+        const wrongAddress = Wallet.createRandom()
+          .address.toString()
+          .slice(0, -1);
+
+        config.util.extendDeep(config, {
+          app: { sponsoredDestinations: [wrongAddress] },
+        });
+        expect(() => verifyServerConfiguration()).to.throw(
+          'Server configuration error:  "app.sponsoredDestinations[0]" failed custom validation because Invalid address:'
+        );
+      });
+
+      it('Should pass if an address is correct', function () {
+        const correctAddress = Wallet.createRandom().address.toString();
+
+        config.util.extendDeep(config, {
+          app: { sponsoredDestinations: [correctAddress] },
+        });
+        expect(() => verifyServerConfiguration()).not.to.throw();
```

Comment:
can you leave a space between each expect.

Reply:
Done